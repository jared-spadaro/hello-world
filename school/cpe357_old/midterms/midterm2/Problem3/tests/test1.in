This sy!
is ea
5
14
