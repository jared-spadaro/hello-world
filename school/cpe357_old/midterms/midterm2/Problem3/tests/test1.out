|This is easy!|
Correct amount of memory allocated
